Makefile

```
@@ -1,20 +1,26 @@
 TARGETS=go1.10 go1.11 go1.12 go1.13 go1.14
 
+build: status=".status.build"
```

I think you only need `.SHELLFLAGS = -ec` this will stop make on the first failure

---

`.ONESHELL :` also can work

---

I think fail fast might not be ideal for this particular use case, what do you think?

---

you know what happens when you allow keeping things broken 😄

---

error should be reported and fail the build always but at the end, so that's what I meant